.github/workflows/ci.yml

```diff
@@ -44,11 +47,22 @@ jobs:
       - name: Install NPM dependencies
         run: npm install
 
-      - name: Build the project
-        run: npm run build
+#      - name: Build the project
+#        run: npm run build
 
-      - name: Run tests
-        run: npm run test
+#      - name: Run tests
+#        run: npm run test
```

Comment:
I assume this is intentional in both the driver projects just now, until the core library PR is merged?